src/apprt/gtk/version.zig

```diff
@@ -7,6 +7,11 @@ const c = @import("c.zig").c;
 /// in the headers. If it is run in a runtime context, it will
 /// check the actual version of the library we are linked against.
 ///
+/// This function should be used in cases where the version check
+/// would affect code generation, such as using symbols that are
+/// only available beyond a certain version. For checks which only
+/// depend on GTK's runtime behavior, use `runtimeAtLeast`.
+///
 /// This is inlined so that the comptime checks will disable the
 /// runtime checks if the comptime checks fail.
 pub inline fn atLeast(
```

Comment:
Can we rename this to `buildAtLeast` or `compiledAtLeast`? Some thing a little more descriptive in the function itself.

Reply:
No because it checks both. I think a general `atLeast` is fine here.

Reply:
Sorry, was off to bed. My original intention was to use `buildAtLeast` and `runtimeAtLeast`. Feel free to fix it in a followup.